source/schema/avro.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 const (
-	avroNS             = "conduit.postgres"
+	avroNS             = ""
```

Comment:
The Conduit SDK is already prefixing every schema name with the connector ID. Do you think that's enough to remove this Avro namespace?

Reply:
Yes, you can remove this.